flink-filesystems/flink-fs-hadoop-shaded/src/main/resources/META-INF/NOTICE

```diff
@@ -32,6 +32,8 @@ This project bundles the following dependencies under the Apache Software Licens
 - org.apache.kerby:kerby-asn1:1.0.1
 - org.apache.kerby:kerby-util:1.0.1
 - org.xerial.snappy:snappy-java:1.1.8.3
+- ch.qos.reload4j:reload4j:1.2.22
+- org.slf4j:slf4j-reload4j:1.7.36
```

Comment:
actually can we just exclude these from the jars? everything should go through log4j2.

Reply:
Done